cmd/secret/cmd/root_test.go

```diff
@@ -51,6 +54,29 @@ func TestRoot_Set(t *testing.T) {
 		}
 		require.EqualValues(t, key, got)
 	})
+	t.Run("expect set data only redis storage", func(t *testing.T) {
+		key := "key value"
+		path := ""
+		redisURL := "localhost:6379"
+		ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+		defer cancel()
+
+		r := New()
+		r.cmd.SetArgs([]string{"set", "--key", key, "--value", "test value", "--cipher-key", "ck", "--redis-url", redisURL})
+		err := r.Execute(ctx)
+		require.NoError(t, err)
+
+		_, err = os.Open(path)
+		require.Error(t, err)
+
+		key = "key"
+		encodedValue := "value"
+
+		rdb := redis.NewClient(&redis.Options{Addr: redisURL, Password: "", DB: 0})
+		s := storage.NewRedisVault(rdb)
+		err = s.SaveData([]byte(key), []byte(encodedValue))
+		require.NoError(t, err)
```

Comment:
Why do you `save`, not `get`? Use `rdb` methods to get value from redis to check that it is actually saved.